### ISSUE TITLE
COPY command gives Access denied

### ISSUE THREAD

**Issue by RoelandR** [AgonConsole8/agon-mos#55]
When I copy a file to an already existing filename the COPY command responses with "Access denied"
It would be great if the response was "Overwrite existing <filename>? (Yes/No)

**Comment by stevesims** [comment]
Yes, currently files are protected from accidental overwriting. This is intentional 

Having the system request permission to overwrite is a sensible enhancement that would be good to add. The main reason why it didn’t make it into the 2.2.x release is just that adding that functionality on top of all the existing changes to the copy command felt a step too far for now. It’s also slightly complicated in that the CLI command and the OS API for copy use the same code, and the API version would need to not include the confirm functionality
